<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add</class>
 <widget class="QDialog" name="Add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>251</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lb_name">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_name">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lb_height">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Height</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_height">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>cm</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="lb_weight">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Weight</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_weight">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>kg</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="lb_age">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Birth Date</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit_birth">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="minimumDateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>1990</year>
            <month>1</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="maximumDate">
           <date>
            <year>2030</year>
            <month>12</month>
            <day>31</day>
           </date>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="lb_mf">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Male/Female</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbx_mf">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>Male</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Female</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btn_save">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SAVE</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
